projects/plugins/jetpack/tests/e2e/specs/blocks/free-blocks.test.js

```diff
@@ -25,147 +24,196 @@ test.describe.parallel( 'Free blocks', () => {
-			expect(
-				await frontend.isRenderedBlockPresent( FormBlock ),
-				'Block should be displayed'
-			).toBeTruthy();
-		} );
-	} );
-
-	test( 'Tiled Gallery block', async ( { page } ) => {
-		await test.step( 'Can visit the block editor and add a Tiled Gallery block', async () => {
-			const blockId = await blockEditor.insertBlock(
-				TiledGalleryBlock.name(),
-				TiledGalleryBlock.title()
-			);
-			const block = new TiledGalleryBlock( blockId, page );
-			await block.addImages();
-
-			await blockEditor.openSettingsSidebar();
-			await block.linkToAttachment();
-		} );
-
-		await test.step( 'Can publish a post with a Tiled Gallery block', async () => {
-			await blockEditor.selectPostTitle();
-			await blockEditor.publishPost();
-			await blockEditor.viewPost();
-		} );
-
-		await test.step( 'Can assert that Tiled Gallery block is rendered', async () => {
-			const frontend = await PostFrontendPage.init( page );
-			expect(
-				await frontend.isRenderedBlockPresent( TiledGalleryBlock ),
-				'Block should be displayed'
-			).toBeTruthy();
-		} );
-	} );
-
-	test( 'Subscribe block', async ( { page } ) => {
-		await prerequisitesBuilder( page ).withActiveModules( [ 'subscriptions' ] ).build();
-
-		await test.step( 'Can visit the block editor and add a Subscribe block', async () => {
-			const blockId = await blockEditor.insertBlock(
-				SubscribeBlock.name(),
-				SubscribeBlock.title()
-			);
-			const block = new SubscribeBlock( blockId, page );
-			await block.checkBlock();
-		} );
-
-		await test.step( 'Can publish a post with a Subscribe block', async () => {
-			await blockEditor.selectPostTitle();
-			await blockEditor.publishPost();
-			await blockEditor.viewPost();
-		} );
-
-		await test.step( 'Can assert that Subscribe block is rendered', async () => {
-			const frontend = await PostFrontendPage.init( page );
-			expect(
-				await frontend.isRenderedBlockPresent( SubscribeBlock ),
-				'Block should be displayed'
-			).toBeTruthy();
+	test.describe( 'Site editor', () => {
+		let siteEditor;
+
+		test.beforeEach( async ( { page } ) => {
+			await test.step( 'Visit site editor page', async () => {
+				siteEditor = await SiteEditorPage.visit( page );
+				await siteEditor.edit();
+				await siteEditor.clearCustomizations();
+			} );
+		} );
+
+		test.afterEach( async () => {
+			await test.step( 'Visit site editor page', async () => {
+				await siteEditor.page.bringToFront();
+			} );
+		} );
+
+		test( 'Subscribe block', async ( { page } ) => {
+			await prerequisitesBuilder( page ).withActiveModules( [ 'subscriptions' ] ).build();
+			let block;
+
+			await test.step( 'Can insert a Subscribe block', async () => {
+				const blockId = await siteEditor.insertBlock(
+					SubscribeBlock.name(),
+					SubscribeBlock.title()
+				);
+				block = new SubscribeBlock( blockId, page );
+				await block.checkBlock();
+			} );
+
+			let newTab;
+			await test.step( 'Can publish a page with a Subscribe block', async () => {
```

Comment:
Maybe we should update this wording, since we're not publishing a page per-se. That's not a blocker though, we can do that in a follow-up PR.

Reply:
Good point. I'll update the wording in a follow-up. I have some more tests and steps that need renaming in mind.